package/package.json

```diff
@@ -49,39 +49,11 @@
 	"type": "module",
 	"peerDependencies": {
 		"@astrojs/db": "^0.9.0",
-		"@vitejs/plugin-react": "^4.2.1",
-		"astro": "^4.4.1",
-		"preact": "^10.19.4",
-		"react": "^18.2.0",
-		"react-dom": "^18.2.0",
-		"solid-js": "^1.8.15",
-		"svelte": "^4.2.11",
-		"vue": "^3.4.19"
+		"astro": "^4.4.1"
```

Comment:
It's `^4.7` now right?

Reply:
Shall I update in this PR? I didn't change the version tbh

Reply:
Well, we can just remove the functionality without changing the minimal Astro version.

If anyone wants to update to the core utility then _they_ will need to update Astro as well

Reply:
Yes I agree with the latter